aws/scripts/py/cache_invalidation.py

```
@@ -0,0 +1,343 @@
+                    "Step %d/%d: %s",
+                    step,
+                    len(environments_to_process),
+                    environment.value,
+                )
+                distribution = distributions[environment]
+                self._invalidate_environment(distribution, environment)
+
+            self.logger.info("Blue-green cache invalidation completed successfully")
+
+        except Exception as e:
+            self.logger.exception("Cache invalidation failed: %s", e)
```

Ruff (0.11.9)

Redundant exception object included in `logging.exception` call

(TRY401)